config/config.go

```diff
@@ -100,17 +101,82 @@ type Config struct {
 	StmtSummary         StmtSummary `toml:"stmt-summary" json:"stmt-summary"`
 }
 
+// nullableBool defaults unset bool options to unset instead of false, which enables us to know if the user has set 2
+// conflict options at the same time.
+type nullableBool struct {
+	Valid bool
```

Comment:
how about:
* s/Valid/IsNull/
* s/Value/IsTrue/

Reply:
Sorry but by default we want the two fields to be false (in case the user does not put anything in the configuration file) and result in `Unset`, if you want I could make it s/Value/NotNull to be compatible with the default value.

Reply:
Will `isValid` better?